src/main/java/org/alfresco/repo/event2/EventConsolidator.java

```diff
@@ -265,6 +274,11 @@ private NodeResource buildNodeResourceBeforeDelta(NodeResource after)
             builder.setAspectNames(aspectsBefore);
         }
 
+        if (nodeTypeBefore != null)
+        {
+            builder.setNodeType(nodeTypeBefore.toPrefixString());
```

Comment:
`NodeResourceHelper` class has a private method called `toString(QName k)` which uses the namespace service to get the prefix string. It makes sense to rename that method (e.g. `getQNamePrefixString`) and make it public so you can use it here instead of `nodeTypeBefore.toPrefixString()`.

Reply:
Done. Thanks Jamal :)